src/coprocessor/dag/rpn_expr/mod.rs

```diff
@@ -19,6 +19,9 @@ use self::impl_op::*;
 use crate::coprocessor::codec::data_type::*;
 use crate::coprocessor::Result;
 
+/// Parses the signed / unsigned int in children, and returns the proper Comparer for the children.
+///
+/// Children represents the left expr and the right expr, so children.len() should be two.
```

Comment:
The implementation detail doesn't need to be explained in the function documentation